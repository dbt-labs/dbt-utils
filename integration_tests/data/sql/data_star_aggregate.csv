group_field_1,group_field_2,value_field
a,b,1
a,b,2
c,d,3
c,e,4