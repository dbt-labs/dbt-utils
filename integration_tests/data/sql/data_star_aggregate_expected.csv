group_field_1,group_field_2,value_field_sum
a,b,3
c,d,3
c,e,4